android/app/src/main/AndroidManifest.xml

```diff
@@ -8,7 +8,8 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        android:usesCleartextTraffic="true">
```

Comment:
This is [apparently required](https://github.com/diachedelic/capacitor-blob-writer?tab=readme-ov-file#android) by `capacitor-blob-writer`, although I hadn't come across any issues while testing.

Reply:
I think it can also be defined by the `capacitor.config.ts` `server.cleartext` property, I have come across this issue myself before so makes sense to pre-emptively enable (capacitor says also required for live-reload from android api 28)
https://capacitorjs.com/docs/config